### ISSUE TITLE
Add keyboard shortcut for sending comment

### ISSUE THREAD

**Issue by virgile-dev** [suitenumerique/docs#1719]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
As a user I need to click on the button to send my comment. which is quite inconvenient and uneffective.

**Describe the solution you'd like**
I should be able to send my comment with `ctrl + enter` 

**Describe alternatives you've considered**

**Discovery, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this and possibly write out some documentation (if applicable).
Maybe add a screenshot or design?

**Do you want to work on it through a Pull Request?**

**Comment by Wormav** [comment]
Hi @AntoLC  and @virgile-dev,

I noticed that this issue is currently unassigned, and I'd like to work on it. Could you confirm that it is still available and whether Cmd+Enter should also be supported on macOS?

Once confirmed, I'll implement the shortcut and add the relevant frontend tests.

**Comment by virgile-dev** [comment]
Hey ! 
Thanks offering your help. 
This is issue touches BlockNote. 
I've created this discussion on their repo some time ago : https://github.com/TypeCellOS/BlockNote/discussions/2669
Let's see if the maintainers are willing to accept the contribution
